manifests/d/DonationCoder/ScreenshotCaptor/4.36.1/DonationCoder.ScreenshotCaptor.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: DonationCoder.ScreenshotCaptor
+PackageVersion: 4.36.1

```

Comment:
```suggestion
PackageVersion: 4.36.2
```
I'm seeing 4.36.2 here:
![image](https://user-images.githubusercontent.com/11600822/120538176-18bb0d80-c3de-11eb-97d3-2d15ab99e933.png)

Output from the application:
![image](https://user-images.githubusercontent.com/11600822/120538317-41db9e00-c3de-11eb-8258-5452c2538cef.png)